index.js

```diff
@@ -107,6 +107,7 @@ if (process.stdout.isTTY) {
 
   let committedGitFiles = [];
   if (isdiffBranchExisted) {
+    log(success('Great, we find the ' + diffBranch + ' branch.'));
```

Comment:
Unfortunately, I have merged this PR without noticing this line. I meant to add the log in debug mode. It's my fault that I didn't communicate well. I'm sorry for the inconvenience.

If you don't mind, could you raise another PR with a log message which prints only in debug mode?

If the base branch is found, you can put a log in debug mode. (something like `Base branch exists`)
If not found, a log can be printed in normal mode. (something like below)

⚠️`The base branch(branch_name) doesn't exist`
`Hence all files will be processed`

Reply:
Well,that's my fault.I will fix it later.